LiskHQ/lisk-sdk
# npx mocha doesn't use mocha.opts file - Closes #3413

### What was the problem?

Running mocha through npx didn't work

### How did I solve it?

- By linking the `mocha.opts` file to the `/test/mocha.opts` folder where mocha is searching for.
- by updating lisk-mocha-runner 

Another options I tested but do not work with `npx`

- to remove mocha.opts (deprecated but still supported by Mocha) and replace with `.mocharc`

### How to manually test it?

in `framework/` run: 
- mocha test/mocha/unit/controller/helpers/validator.js
- npm run mocha:unit test/mocha/unit/controller/helpers/validator.js
- npx mocha test/mocha/unit/controller/helpers/validator.js
and all should be able to run that test or any other test

### Review checklist

- [ ] The PR resolves #3413
